packages/slice-machine/src/features/customTypes/customTypesBuilder/GenerateSliceWithAiModal/SliceCard.tsx

```diff
@@ -20,7 +20,7 @@ export function SliceCard(props: SliceCardProps) {
     slice.status === "success";
 
   return (
-    <Card disabled={loading} style={{ width: 394 }}>
+    <Card disabled={loading} size="medium">
```

Comment:
if the card size depends on the layout of the parent (grid with two columns), why do we need to specify the size here?

Reply:
It's about the height. And we don't actually need to specify it because "medium" is the default. I removed it ☑️